custom_model_runner/datarobot_drum/drum/gpu_predictors/nemo_predictor.py

```diff
@@ -94,6 +115,41 @@ def supported_payload_formats(self):
     def has_read_input_data_hook(self):
         return False
 
+    def _get_custom_artifacts(self):
+        code_dir_abspath = os.path.abspath(self._code_dir)
+
+        custom_py_paths = list(Path(code_dir_abspath).rglob("{}.py".format(CUSTOM_FILE_NAME)))
+        remote_artifact_paths = list(Path(code_dir_abspath).rglob(REMOTE_ARTIFACT_FILE_EXT))
+
+        if len(custom_py_paths) + len(remote_artifact_paths) > 1:
+            error_mes = (
+                "Multiple custom.py/.remote files were identified in the code directories sub directories.\n"
+                "The following custom model files were found:\n"
+            )
+            error_mes += "\n".join(
+                [str(path) for path in (custom_py_paths + remote_artifact_paths)]
+            )
+            self.logger.error(error_mes)
+            raise DrumCommonException(error_mes)
+
+        return custom_py_paths, remote_artifact_paths
+
+    def _load_model_with_custom_hooks(self, params):
+        try:
+            model_adapter = PythonModelAdapter(
+                model_dir=self._code_dir, target_type=self.target_type
+            )
+            sys.path.append(self._code_dir)
+            model_adapter.load_custom_hooks()
+            # disable the check of scoring hook existence
+            model_adapter._custom_hooks[CustomHooks.SCORE] = True
```

Comment:
I don't see the full picture, but the expected value is a method rather than a boolean. Please validate that it does not break things up.

Reply:
validated, this hook is never invoked